javascript-functions-practice/get-length-of-array.js

```diff
@@ -0,0 +1,3 @@
+/* exported getLengthOfArray */
+const getLengthOfArray = (array = ['foo', 'bar', 'baz']) => array.length;
```

Comment:
You shouldn't be setting default values for your array in your parameter. You should just let the test case pass in the array